### PR TITLE
CHORE standalone GraphQL Type Definition files for `graphql-codegen`

### PR DESCRIPTION
**Pull request by cantremember**

### CODE REVIEW

File: src/graphql/core.types.ts [hunk 1]
```diff
@@ -16,7 +16,7 @@ const REGEX_FORMAT = /^\d{4}-\d{2}-\d{2}$/
 //   @see `scalar Timestamp`
 const FORMAT_TIMESTAMP = 'YYYY-MM-DD[T]HH:mm:ss.SSSZ';
 
-const coreTypeDefsString = /* GraphQL */ `
+export const coreTypeDefs = gql`
```
Comment (mcorrigan89) [inline src/graphql/core.types.ts:19]: Can we keep this the way it is? Need for some of the generation files, per: https://github.com/joylifeinc/server-core/pull/109

Comment (cantremember) [reply src/graphql/core.types.ts:19]: we can *totally* do that

**Review by nshikov — APPROVED** [review]
@cantremember i can confirm that codegen worked in my case with those typedefs.js files, so can approve
no sure about @mcorrigan89 case tho

File: src/graphql/core.types.ts (hunk 1 above)
Comment (cantremember) [reply src/graphql/core.types.ts:19]: ah -- okay, now that i understand that `display_service` is using the same tooling, i have confirmed that referenceing these new files will produce the proper results

https://github.com/joylifeinc/display_service/pull/2 <= here's the PR

**Pull request merged**